web/sites/default/settings.php

```diff
@@ -125,6 +125,10 @@
 else if($env == 'live') {
   $config['search_api.server.searchstax']['backend_config']['connector_config']['core'] = 'Production';
 }
+else if($env == 'lando') {
+  $config['search_api.server.searchstax']['backend_config']['connector_config']['host'] = 'solr';
+  $config['search_api.server.searchstax']['backend_config']['connector_config']['core'] = 'index';
```

Comment:
I don't think you need these manual overrides here since you've got a config split for lando local environments. You should be able to just put these values directly in web/sites/default/config_local/search_api.server.searchstax.yml

Reply:
Removed the host name override but keeping the core override. The key file appears to override the Core name.

Reply:
Hmm. you're right. But the key file also overrides the host so now I'm getting an error that Local Solr is unreachable and it's trying to reach http://ss626374-s4p42op1-us-west-2-aws.searchstax.com:8983/solr/#/Local. Is there a way to disable the key overrides in the local config split?

Reply:
Ah, that would require we create another key file like "solr_host_local.key" and add a config split for the key setting that points to that file. Similar how we did it for solr_core_prod.key

Reply:
Yeah, I suppose that would work. Or you could just revert back to overriding both in settings.php. I'll let you decide which route you think is better.

Reply:
Updated settings.php to set "host" directly.